it/mail/chrome/messenger/messengercompose/mailComposeEditorOverlay.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY attachImageSource.label         "Allega questa immagine al messaggio">
 <!ENTITY attachImageSource.accesskey     "s">
 
-<!ENTITY attachLinkSource.label          "Allega il sorgente di questo collegamento al messaggio">
+<!ENTITY attachLinkSource.label          "Allega la sorgente di questo link al messaggio">
```

Comment:
CORRECTIONS NEEDED. It should be as follows ("sorgente" in this very meaning is masculine): 

+<!ENTITY attachLinkSource.label          "Allega il sorgente di questo link al messaggio">